### PR TITLE
Sequence/collection cards can open in new tab

### PR DESCRIPTION
**Pull request by jimrandomh**

### CODE REVIEW

**Comment by jimrandomh** [comment]
Fixes #1967.

File: packages/lesswrong/components/sequences/SequencesGridItem.jsx [hunk 1]
```diff
@@ -83,6 +78,7 @@ const styles = theme => ({
   },
 
   bottom: {
+    pointerEvents: "none",
```
Comment (Raemon) [inline packages/lesswrong/components/sequences/SequencesGridItem.jsx]: Why is this necessary? (I recall you talking about it with Oli and him being skeptical of pointer-event settings. I'm less skeptical of pointer event settings but also unsure why it's needed here, given that it wasn't needed in any of the other LinkCard components)

Comment (jimrandomh) [reply packages/lesswrong/components/sequences/SequencesGridItem.jsx]: Removed. It wasn't actually necessary; it was the remnant of a previous way of doing this that I tried, which wasn't the best way. (It also wasn't harmful, in this case, since nothing inside `classes.bottom` needed to be clickable.)
